websites/M/Modrinth/presence.ts

```diff
@@ -8,17 +8,17 @@ presence.on("UpdateData", async () => {
 			largeImageKey: "https://i.imgur.com/GnMSCG4.png",
 			startTimestamp: browsingTimestamp,
 		},
-		{ pathname, hostname, href } = document.location,
+		{pathname, hostname, href} = document.location,
 		pathSplit = pathname.split("/").filter(x => x),
 		active = document.querySelector('[class*="is-active"]');
 
 	switch (hostname) {
 		case "modrinth.com": {
 			if (active?.textContent) {
-presenceData.details = `Browsing ${
+				presenceData.details = `Browsing ${
 					pathSplit[0]
 				} ${active?.textContent.toLowerCase()}`;
-}
+			}
```

Comment:
```suggestion
presence.setActivity(presenceData); 
return;
			}
```

Reply:
I think it's a little bit not what we expected
![image](https://user-images.githubusercontent.com/47828778/206904810-52d2c569-b32b-46c1-8e00-4c33995fe383.png)
![image](https://user-images.githubusercontent.com/47828778/206904842-be936690-e88d-4ed0-b76c-3b85c03efeb6.png)

Reply:
> I think it's a little bit not what we expected ![image](https://user-images.githubusercontent.com/47828778/206904810-52d2c569-b32b-46c1-8e00-4c33995fe383.png) ![image](https://user-images.githubusercontent.com/47828778/206904842-be936690-e88d-4ed0-b76c-3b85c03efeb6.png)

oh oops.